packages/jest-cli/src/__tests__/watch.test.js

```diff
@@ -95,6 +97,7 @@ describe('Watch mode flows', () => {
       pipe,
       new TestWatcher({isWatchMode: true}),
```

Comment:
it's on my plan! bun not in this PR :)
I already started refactoring the CLI flow